torch/_inductor/fx_passes/post_grad.py

```diff
@@ -682,12 +682,15 @@ def can_replace(node, mutated_arg):
 
     inplaceable_ops = {
         aten.index_put.default: InplaceableOp(aten.index_put_.default, 0),
+        aten._unsafe_index_put.default: InplaceableOp(
+            aten._unsafe_index_put_.default, 0
+        ),
```

Comment:
Just a side comment, but do we have any guarantees that the rest of the inductor stack knows how to handle in-place ops at all?

Reply:
Yes, the inductor stack post fx passes supports in place operations. However, the graph passes don’t. Check fx_oasses/readme for the  invariants